.github/workflows/cpp-packaging.yml

```diff
@@ -30,9 +30,9 @@ env:
   demumbleVer: "1.1.0"
   # Use SHA256 for hashing files.
   hashCommand: "sha256sum"
-  # Xcode version 11.7 is the version we build the SDK with.
+  # Xcode version 12 is the version we build the SDK with.
   # Our MacOS runners will use the version in /Applications/Xcode_${xcodeVersion}.app
-  xcodeVersion: "11.7"
```

Comment:
This changes our Xcode minimum requirement from 11.7 to 12, so we should discuss it before making this change.

Does xcode 11 not support xcframeworks at all?

We should check what the minimum Xcode version is for the Firebase iOS SDK as well.

Reply:
Xcode 11 support xcframeworks. But only 12+ support simulator-arm64.

Reply:
We need to build xcframeworks with xcode 12+. But it is possible to build our integration_test.app with Xcode 11.7 (I tried locally).
Also checked the minimum Xcode version for the Firebase iOS SDK. It's Xcode 12.0 (or later).
https://github.com/firebase/firebase-ios-sdk#development

Reply:
OK, we should bump our minimum listed requirement to 12 then, and ensure that all of our workflows use the same version.